docs/concepts.rst

```diff
@@ -359,6 +359,24 @@ we can disable persistent caching altogether:
 If your goal is just to force an entity to be recomputed more frequently, you may
 want :ref:`@changes_per_run <changes_per_run>` instead.
 
+Persistent caching can also be globally disabled:
+
+.. code-block:: python
+
+    builder.set('core__persist_by_default', False)
+
+This only changes the default behavior, so it can be explicitly re-enabled for
```

Comment:
Do we have any documentation that explains the consequences of turning off both persistence and memoization?

Reply:
I don't think so. Let's talk on slack what we can add for it.